src/components/Dropdown/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState, useCallback } from "react";
+
+export default function Dropdown(props) {
+	const [isExpanded, setIsExpanded] = useState(false);
+  const {label, dropdown, items} = props;
+	const expandMenu = useCallback(() => {
+    setIsExpanded(s => !s);
+  }, [setIsExpanded]);
+
+  return (
+  	<div className="dropdown">
+      <label>{items.label}</label><br />
```

Comment:
You are trying to access a particular property of an object that is inside an array - it won't work and for the dropdown label you should add a separate property.